src/main/java/mate/academy/service/UserService.java

```diff
@@ -8,13 +9,23 @@ public class UserService {
             new User("alice@i.ua", "1234")
     };
 
+    public static User[] getUsers() {
+        return users;
+    }
+
```

Comment:
Actually, we do not need this getter here. We should use only `findByEmail` method, when doing login.